bin/improver-temp-lapse-rate

```diff
@@ -94,14 +102,24 @@ def main():
     orography_cube = load_cube(args.orography_filepath)
     land_sea_mask_cube = load_cube(args.land_sea_mask_filepath)
 
-    result = LapseRate(
-        max_height_diff=args.max_height_diff,
-        nbhood_radius=args.nbhood_radius,
-        max_lapse_rate=args.max_lapse_rate,
-        min_lapse_rate=args.min_lapse_rate).process(temperature_cube,
-                                                    orography_cube,
-                                                    land_sea_mask_cube)
-
+    if args.return_dalr:
+        result = temperature_cube.copy(
+            data=np.full_like(temperature_cube.data, U_DALR.points[0]))
+        result.rename('temperature_lapse_rate')
+        result.units = U_DALR.units
+        result.attributes['source'] = 'IMPROVER'
+        result.attributes['institution'] = 'Met Office'
```

Comment:
Should this just be copied from the input cube?

Reply:
No.  We are creating a new cube with new data.  The source is IMPROVER and the institution is the Met Office (although the second is less obvious and I can remove that attribute for now if you would prefer).  Discussions are currently in progress and you are welcome to check this with @bjwheltor

Reply:
Will remove "institution" from this bit of code (not from the cube), history TBD.